pom.xml

```diff
@@ -205,6 +205,11 @@
               </goals>
             </execution>
           </executions>
+        <configuration>
```

Comment:
Note: this has already been addressed in BEAM via https://github.com/apache/incubator-beam/commit/119812a66bceb9e8b91dcd7afb26bc9b7ee5eee7